webviz_config/_config_parser.py

```diff
@@ -194,8 +194,8 @@ def _call_signature(
     if "webviz_settings" in argspec.args:
         special_args += "webviz_settings=webviz_settings, "
 
-    return (
-        f"{plugin_name}({special_args}**{kwargs})",
+    return  (
```

Comment:
I am not very happy with this, but unsure of a good solution here - not sure if it is worth e.g. looping over the kwargs to more explicitly special-case Paths, or if this is good enough - especially given the long term plans of webviz

Reply:
We will probably move `webviz-config` over to maintenance mode going forward (since the FMU usage pattern, funding this repo, is changing), and instead focus on collaborating with common React (and corresponding Dash) components.

I.e. I think your fix is reasonable given that context. 👍